### PR TITLE
(maint) PowerShell Formatting

### PR DESCRIPTION
**Pull request by TheCakeIsNaOH**
## Description Of Changes

PowerShell formatting changes according to PSSA settings in chocolatey/Chocolatey.Cake.Recipe#73. 

## Motivation and Context

Brings the PowerShell scripts to a consistent formatting baseline in this repository to prepare for #2703.

The `chocolateyInstall.ps1` in the `zip.template` and `msi.template` packages included for the pester tests will give parsing errors, these will need to be excluded. The parsing errors are due to the tokens that are to be replaced when the template is used.

The `tools` directory also need to be excluded as it contains PowerShell scripts from various Cake modules/tools/addins/etc that should not be checked.

## Testing

Ran `Invoke-ScriptAnalyzer -Path . -Recurse -Settings "\path\to\Chocolatey.Cake.Recipe\Chocolatey.Cake.Recipe\Content\formatting-settings.psd1"` to ensure that the PowerShell formatting is acceptable according to the agreed on settings.
Ran nunit and pester tests to ensure that nothing was broken.

## Change Types Made

* [ ] Bug fix (non-breaking change)
* [ ] Feature / Enhancement (non-breaking change)
* [ ] Breaking change (fix or feature that could cause existing functionality to change)
* [x] PowerShell code changes.
* [x] Maint

## Related Issue

Formatting changes for #2703
Part of #13

## Change Checklist

* [ ] Requires a change to the documentation
* [ ] Documentation has been updated
* [ ] Tests to cover my changes, have been added
* [ ] All new and existing tests passed.
* [x] PowerShell v2 compatibility checked.

### CODE REVIEW

**Review by vexx32 — DISMISSED** [review]
This looks good to me! A few comments here and there. There's more we could do with these scripts going forward, but this is a great first step.

Thanks for the PR! 💖 😊

File: src/chocolatey.resources/helpers/ChocolateyTabExpansion.ps1 [hunk 1]
```diff
@@ -45,31 +47,31 @@ $proUninstallOptions = " --from-programs-and-features"
...
+    optimize  = "--deflate-nupkg-only --id='' -?" + $allcommands
+    export    = "--include-version-numbers --output-file-path='' -?" + $allcommands
+    template  = "--name=''" + $allcommands
 }
 
 try {
-  # if license exists
-  # add in pro/biz switches
+    # if license exists
+    # add in pro/biz switches
```
[the remark this answers is not in the document]
Comment (vexx32) [reply src/chocolatey.resources/helpers/ChocolateyTabExpansion.ps1:74]: @gep13 is this something we want/need to file an issue for?

[the remark this answers is not in the document]
Comment (TheCakeIsNaOH) [reply src/chocolatey.resources/helpers/ChocolateyTabExpansion.ps1:74]: Issue opened here: https://github.com/chocolatey/choco/issues/2887

File: GenerateDocs.ps1 [hunk 2]
```diff
@@ -373,49 +394,56 @@ function Generate-CommandReference($commandName, $order) {
...
-    }else {
-      $FormattedName = $SplitName[0].ToLower() + ($SplitName[1] -creplace '[A-Z]', '-$&').ToLower()
+    if (Test-Path($docsFolder)) {
+        Remove-Item $docsFolder -Force -Recurse -EA SilentlyContinue
+    }
+    if (-not(Test-Path $docsFolder)) {
+        mkdir $docsFolder -EA Continue | Out-Null
+    }
+    if (-not(Test-Path "$docsFolder\create\functions")) {
+        mkdir "$docsFolder\create\functions" -EA Continue | Out-Null
```
[the remark this answers is not in the document]
Comment (TheCakeIsNaOH) [reply GenerateDocs.ps1]: Fixed

File: GenerateDocs.ps1 [hunk 3]
```diff
@@ -373,49 +394,56 @@ function Generate-CommandReference($commandName, $order) {
...
-      $FormattedName = "get-os-architecture-width"
-    } elseif($_.Name -eq 'Get-UACEnabled') {
-      $FormattedName = "get-uac-enabled"
-    }else {
-      $FormattedName = $SplitName[0].ToLower() + ($SplitName[1] -creplace '[A-Z]', '-$&').ToLower()
+    if (Test-Path($docsFolder)) {
+        Remove-Item $docsFolder -Force -Recurse -EA SilentlyContinue
+    }
+    if (-not(Test-Path $docsFolder)) {
+        mkdir $docsFolder -EA Continue | Out-Null
```
[the remark this answers is not in the document]
Comment (TheCakeIsNaOH) [reply GenerateDocs.ps1]: Fixed

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@TheCakeIsNaOH thanks for getting all of these fixed up!

**Pull request merged**